debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/BinlogReader.java

```diff
@@ -378,18 +399,54 @@ protected void handleServerHeartbeat(Event event) {
     /**
      * Handle the supplied event that signals that an out of the ordinary event that occurred on the master. It notifies the slave
      * that something happened on the master that might cause data to be in an inconsistent state.
-     * 
+     *
      * @param event the server stopped event to be processed; may not be null
      */
     protected void handleServerIncident(Event event) {
-        logger.trace("Server incident: {}", event);
+        if (event.getData() instanceof EventDataDeserializationExceptionData) {
+            EventDataDeserializationExceptionData data = event.getData();
+
+            // TODO make it an option
+            EventDeserializationFailureHandlingMode deserializationFailureHandling = EventDeserializationFailureHandlingMode.FAIL;
```

Comment:
`deserializationFailureHandling` is always `FAIL`... are we missing where that's actually set somewhere? Should it be set via a new configuration property?

Reply:
Yes, that's what I meant by

> The option to control the behavior is still missing

It will be set via that new option.

Reply:
The option is there now, defaulting to `FAIL`, i.e. the current behavior.